scripts/build_fuzzers.py

```diff
@@ -0,0 +1,65 @@
+import subprocess
+import toml
+import typing
```

Comment:
remove, is unused. Use this an opportunity to remove the imports that are not used.